modules/playlists/src/main/java/org/opencastproject/playlists/PlaylistRestService.java

```diff
@@ -0,0 +1,600 @@
+      + "            {\n"
+      + "                \"contentId\": \"ID-about-opencast\",\n"
+      + "                \"type\": \"EVENT\"\n"
+      + "            },\n"
+      + "            {\n"
+      + "                \"contentId\": \"ID-3d-print\",\n"
+      + "                \"type\": \"EVENT\"\n"
+      + "            }\n"
+      + "        ],";
+
+  public static final String SAMPLE_PLAYLIST_XML = "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?><"
+      + "ns3:playlist xmlns:ns2=\"http://mediapackage.opencastproject.org\" "
+      + "xmlns:ns3=\"http://playlist.opencastproject.org\" id=\"1059\"><organization>mh_default_org</organization>"
+      + "<entries id=\"1061\"><contentId>ID-av-portal</contentId><type>EVENT</type></entries><entries id=\"1062\">"
+      + "<contentId>ID-av-print</contentId><type>EVENT</type></entries><title>Opencast Playlist</title>"
+      + "<description>This is a playlist about Opencast</description><creator>Opencast</creator>"
+      + "<updated>1701787700848</updated><accessControlEntries><allow>true</allow><role>ROLE_USER_BOB</role>"
+      + "<action>read</action></accessControlEntries></ns3:playlist>";
+
+  public static final String SAMPLE_PLAYLIST_ENTRIES_XML =
+      "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>\n"
+      + "<entries>\n"
+      + "\t<entry id=\"1061\">\n" + "\t\t<contentId>ID-av-portal</contentId>\n" + "\t\t<type>EVENT</type>\n"
+      + "\t</entry>\n"
+      + "\t<entry id=\"1062\">\n" + "\t\t<contentId>ID-av-print</contentId>\n" + "\t\t<type>EVENT</type>\n"
+      + "\t</entry>\n" + "</entries>";
+
+  /** The playlist service instance */
+  private PlaylistService service;
+
+  /** The search service */
+  protected SearchService searchService = null;
+
+  /** The authorization service */
+  protected AuthorizationService authorizationService = null;
+
+  /**
+   * Sets the playlist service
+   *
+   * @param service
+   *          the playlist service instance
+   */
+  @Reference
+  public void setService(PlaylistService service) {
+    this.service = service;
+  }
+
+  @Reference
+  protected void setSearchService(SearchService searchService) {
+    this.searchService = searchService;
+  }
+
+  @Reference
+  public void setAuthorizationService(AuthorizationService authorizationService) {
+    this.authorizationService = authorizationService;
+  }
+
+  @GET
+  @Produces(MediaType.APPLICATION_JSON)
+  @Path("playlist.json")
+  @RestQuery(
+      name = "playlist",
+      description = "Get a playlist.",
+      returnDescription = "A playlist as JSON",
+      restParameters = {
+          @RestParameter(name = "id", isRequired = true, description = "The playlist identifier", type = STRING),
+          @RestParameter(name = "withPublications", isRequired = false, description = "If available publications for"
+              + "the content should be returned. Only works for content of type EVENT.", type = BOOLEAN,
+              defaultValue = "true")
```

Comment:
```suggestion
          @RestParameter(name = "withPublications", isRequired = false, description = "If available publications for "
              + "the content should be returned. Only works for content of type EVENT.", type = BOOLEAN,
              defaultValue = "true")
```

Missing space. But preferably, this should be formatted as.

```suggestion
          @RestParameter(
              name = "withPublications", 
              isRequired = false,
              description = "If available publications for the content should be returned. "
                  + "Only works for content of type EVENT.", 
              type = BOOLEAN,
              defaultValue = "true"
          )
```